packages/ui/src/types.ts

```diff
@@ -23,3 +23,23 @@ export interface TagSelectOption {
   value: Tag;
   label: string;
 }
+
+export const alertTypes = [
```

Comment:
are all of these alerts related to Smooth? Maybe we could rename this to `smoothAlertTypes`

Reply:
Not all of them, (successAlert, errorAlert, onlyEditableFeesAlert, feeAlreadySetToAllAlert) are related to the process of changing the FR. Those can appear without checking the join Smooth switch. Would be better to split it in two? (generalAlertTypes and smoothAlertTypes)